### DISCUSSION TITLE
ignore

### DISCUSSION THREAD

**Discussion by stvzldn** [mytourbook/mytourbook#1040] [General]

**Comment by FJBDev** [comment]
You need to ensure that you have the 23.3.1 URL, in your screenshots, you have 22.2.1

![image](https://user-images.githubusercontent.com/25235665/227647410-143369a8-305f-440d-a20f-8e7a707f8598.png)

**Reply by stvzldn** [reply]
I saw that, I had followed the prompts in the wiki too closely!
All fine now.
